### DISCUSSION TITLE
can't generate card， console prompt：this.field_names is not iterable

### DISCUSSION THREAD

**Discussion by lily8186** [ObsidianToAnki/Obsidian_to_Anki#427] [Q&A]
obsidian to anki can't generate card
Only the connection to ANKI is displayed, but the ANKI card will not be generated. 
console prompt:
Uncaught (in promise) TypeError: this.field_names is not iterable
The prompt is shown in the figure.please help me
![控制台信息](https://github.com/Pseudonium/Obsidian_to_Anki/assets/76209421/300e913c-b4f5-4ba5-a92e-754477f0bce5)
![插件设置](https://github.com/Pseudonium/Obsidian_to_Anki/assets/76209421/f942a31d-b192-4dc8-af09-0ca060933337)
![提示信息](https://github.com/Pseudonium/Obsidian_to_Anki/assets/76209421/145e72d3-1c88-4397-9891-6747e84547df)

**Comment by LightishRedd** [comment]
Have you resolved this issue in any way? I'm having the same issue as well.
